testframework/src/main/java/net/neoforged/testframework/impl/FrameworkCollectors.java

```diff
@@ -86,15 +87,21 @@ public static List<Test> forMethodsWithAnnotation(ModContainer container, Class<
 
         public static List<Test> forGameTestMethodsWithAnnotation(ModContainer container, Class<? extends Annotation> annotation) {
             return findMethodsWithAnnotation(container, SIDE_FILTER, annotation)
-                    .filter(method -> method.getParameterTypes().length == 1 && method.getParameterTypes()[0].isAssignableFrom(ExtendedGameTestHelper.class))
+                    .filter(method -> method.getParameterTypes().length == 1 && GameTestHelper.class.isAssignableFrom(method.getParameterTypes()[0]))
```

Comment:
This seems to be reversed? `isAssignableFrom` returns `true` if the specified class is the same as the owner or a superclass/interface of it.

Reply:
Previously it was testing if the param type was a parent of `ExtendedGameTestHelper`. Now it's testing if the param type is a child of `GameTestHelper`.